ssh/ssh.go

```diff
@@ -90,9 +92,24 @@ func RemoteSSHCommand(user string, privateKey string, host string, port string,
 		}
 	}
 
-	log.Info("Running command %s", cmd)
-	err = session.Run(cmd)
-	if err != nil {
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
```

Comment:
All those are valid signals that typically indicate the the user wants the process to shutdown.